packages/edit-site/src/components/sidebar-edit-mode/page-panels/page-summary.js

```diff
@@ -32,8 +33,12 @@ export default function PageSummary( {
 			<PluginPostStatusInfo.Slot>
 				{ ( fills ) => (
 					<>
-						<PostFeaturedImagePanel withPanelBody={ false } />
-						<PrivatePostExcerptPanel />
+						<VStack spacing={ 2 }>
```

Comment:
Styling in this Vstack should match this one: https://github.com/WordPress/gutenberg/pull/61235/files#diff-89b172fd62da4f8d2f19034e869e07494dffe29c7bcebf893ae1d4ddcbc2093bR85 for now.